### ISSUE TITLE
A11y - OU: ARIA dialog and alert dialog nodes should have an accessible name

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5475]
Ensure every ARIA dialog and alert dialog node has an accessible name. [+more informationLink opens in a new window+|https://dequeuniversity.com/rules/axe/4.10/aria-dialog-name?application=AxeChrome]

h3. Element Location:

{noformat}#b2b-organization-unit-confirm-modal{noformat}

h3. To solve this problem, you need to fix at least (1) of the following:

* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute

!Screen Shot 2024-11-13 at 16.51.43 PM.png|width=2052,height=1818,alt="Screen Shot 2024-11-13 at 16.51.43 PM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39576**



[created from NEXT-39576, comment 512410]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39576, comment 521038]
